### ISSUE TITLE
eth_feeHistory wrongly returns error Incorrect block

### ISSUE THREAD

**Issue by mshakeg** [hashgraph/hedera-json-rpc-relay#813]
### Description

Occasionally when making `eth_feeHistory` calls the relay returns the error **Incorrect block** such as in the following error response:
```
{
   "reason":"bad response",
   "code":"SERVER_ERROR",
   "status":400,
   "headers":{
      "date":"Wed, 11 Jan 2023 10:29:29 GMT",
      "content-type":"application/json; charset=utf-8",
      "content-length":"200",
      "connection":"close",
      "access-control-allow-origin":"*",
      "access-control-allow-methods":"GET,HEAD,PUT,POST,DELETE",
      "via":"1.1 google",
      "cf-cache-status":"DYNAMIC",
      "server":"cloudflare",
      "cf-ray":"787cf878ecf68023-IAD"
   },
   "body":{
      "jsonrpc":"2.0",
      "id":3799,
      "error":{
         "code":-32000,
         "name":"Incorrect block",
         "message":"[Request ID: 2c192c09-7422-4204-a87e-665b2e09ba79] Request beyond head block: requested 29328315, head 29328309"
      }
   },
   "requestBody":{
      "method":"eth_feeHistory",
      "params":[
         "0x4",
         "latest",
         [
            50
         ]
      ],
      "id":3799,
      "jsonrpc":"2.0"
   },
   "requestMethod":"POST",
   "url":"https://testnet.hashio.io/api"
}
```
The block isn't passed explicitly in the `eth_feeHistory` request rather `latest` is as indicated above. Additionally, the error response doesn't add up as at the time of the request i.e. 11 Jan 2023 10:29:29 GMT [Block 29328315](https://hashscan.io/testnet/block/29328315) had already been created and the head block was definitely NOT the earlier block 29328309 as the error message states.

### Steps to reproduce

As stated the error happens occasionally, but I think the above explanation is sufficient evidence of the issue.

### Hedera network

testnet

### Version

hashio-relay/0.15.0

### Operating system

None

**Comment by Kalina-Todorova** [comment]
@mshakeg we will look into this intermittent issue and update you here.

**Comment by mshakeg** [comment]
Hi @natanasow just wanted to ask if the fix for this issue has been included in `relay/0.16.0-rc4` ? As I'm still getting this error.
```json
{
  "reason": "bad response",
  "code": "SERVER_ERROR",
  "status": 400,
  "headers": {
    "date": "Mon, 30 Jan 2023 14:00:52 GMT",
    "content-type": "application/json; charset=utf-8",
    "content-length": "194",
    "connection": "close",
    "access-control-allow-origin": "*",
    "access-control-allow-methods": "GET,HEAD,PUT,POST,DELETE",
    "via": "1.1 google",
    "cf-cache-status": "DYNAMIC",
    "server": "cloudflare",
    "cf-ray": "791abc7f0f11056e-IAD"
  },
  "body": {
    "jsonrpc": "2.0",
    "id": 43,
    "error": {
      "code": -32000,
      "name": "Incorrect block",
      "message": "[Request ID: 930f050e-8f85-450f-8132-66831d01d041] Request beyond head block: requested 232923, head 232922"
    }
  },
  "requestBody": {
    "method": "eth_feeHistory",
    "params": [
      "0x4",
      "latest",
      [
        50
      ]
    ],
    "id": 43,
    "jsonrpc": "2.0"
  },
  "requestMethod": "POST",
  "url": "https://testnet.hashio.io/api"
}
```

**Comment by natanasow** [comment]
@mshakeg The fix is included in `relay/0.17.0-rc1`.
